### ISSUE TITLE
[TODO] Start Date & Bookmarks

### ISSUE THREAD

**Issue by RowanMaBoot** [Altemio/SevenKingdoms-master#5]
We're looking at a start date of 6500. I'm going to get the bookmarks set up, edit the defines and hopefully get the timeline working. Last time I had a crack at it, the mod crashed repeatedly...

Our second bookmark would be prior to the Doom of Valyria, which means a more unified Westeros.

If anyone wants to discuss the chosen date, this is the place to do it.

**Comment by RowanMaBoot** [comment]
Start date has been set. The 1st of March, 6566. This can be tweaked if ya'll want to tweak it, otherwise that is what we've got.

Our starting bookmark is also up; will need a custom name for it, and if you guys want, a secondary bookmark date before the Doom, but close enough that players will see it during a play through (or not if it doesn't happen).

**Comment by tannerflick4** [comment]
So how playable is our version right now? Would it CTD?

**Comment by ZeroFighterR** [comment]
So far I know its playable, but vanilla events will fire and mention areas that dont exist.

**Comment by tannerflick4** [comment]
Warning: Pressing on present a courtier button (commander, noble, etc) results in CTD. Also, hitting "send" for a marriage to initiate it causes a CTD. So, assumption is not playable yet.

**Comment by RowanMaBoot** [comment]
I did not get a crash when using those buttons (including marriage), what culture/religion were you? Could be an issue with one of them.

Edit: Latest version crashes after a few days, regardless of what you do. Hmm. Need to find out what is wrong.

Editx2:

Scratch that. Fixed the crash in this case; was no technology files for the new duchies or renamed duchies (we have to delete the old technology files and completely re-gen history as it does not account for changes).

It is possible the crash you're talking about was happening not due to your issues, but because of that. Was able to marry, introduce some courtiers and what-not into my court fine (as Frodo of the Ring).
